js/goto.js

```diff
@@ -9,10 +12,21 @@ function redirectUserToPage(page) {
 function redirectionHandler(data) {
   if (data.incompleteUserDetails) {
     redirectUserToPage(SIGNUP_URL);
-  } else if (hasVisitedJoin == 'true' || hasVisitedJoin == null) {
-    redirectUserToPage(`${HOME_URL}/join`);
+  } else if (urlParams.get('dev') === 'true') {
+    if (
+      data.roles.developer == true &&
```

Comment:
instead of `date.roles.developer == true` just `data.roles.developer` should suffice. Also, for comparison use `===` not `==`.

Reply:
Thank you @apurvkhare for learning and I am done with changes.